### PR TITLE
Better xml tree generation in mcts example

### PR DESCRIPTION
**Pull request by mrbermell**

### CODE REVIEW

File: examples/mcts_example.py [hunk 1]
```diff
@@ -70,7 +71,7 @@ def simple_heuristic(game: botbowl.Game, agent:botbowl.Agent):
             own += 0.1
         elif game.get_winner() is None:
             opp += 0.1
-    return 0.5 + own - opp
```
Comment (njustesen) [inline examples/mcts_example.py:73]: The heuristic should return a value between 0 and 1 as if it's a prediction of the zero-sum reward at the end of the game. So I think we should keep the 0.5 here.

File: examples/mcts_example.py [hunk 2]
```diff
@@ -125,6 +126,7 @@ def _extract_actions(self, game):
                     actions.append(Action(action_choice.action_type, position=position))
             else:
                 actions.append(Action(action_choice.action_type))
+        random.shuffle(actions)
```
Comment (njustesen) [inline examples/mcts_example.py:129]: Can you make this optional and disabled by default? I want the script to follow the tutorial.

**Review by njustesen — CHANGES_REQUESTED** [review]
Everything regarding the XML looks good but I have a few comments to the rest.

File: examples/mcts_example.py (hunk 1 above)
Comment (mrbermell) [reply examples/mcts_example.py]: Good point, thanks! It's updated!

**Pull request merged**